app/models/accounting/quickbooks/updater.rb

```diff
@@ -111,6 +113,16 @@ def extract_qb_data(txn)
         txn.save!
       end
 
+      def lookup_currency(txn)
+        project = txn.project
+
+        if txn.quickbooks_data && txn.quickbooks_data[:currency_ref]
+          Currency.find_by(code: quickbooks_data[:currency_ref][:value]).try(:id)
+        elsif project
+          Currency.find_by(id: project.currency_id)
```

Comment:
Can't you just use `project.currency` here?

Reply:
I could :)
thanks